packages/next/build/index.ts

```diff
@@ -208,10 +208,26 @@ export default async function build(dir: string, conf = null): Promise<void> {
     }),
   ])
 
+  const clientConfig = configs[0]
+
+  if (
+    clientConfig.optimization &&
+    (clientConfig.optimization.minimize == false ||
+      (clientConfig.optimization.minimizer &&
+        clientConfig.optimization.minimizer.length === 0))
+  ) {
+    console.warn(
+      chalk.bold.yellow(`Warning: `) +
+        chalk.yellow(
```

Comment:
Let's bold this instead of making it yellow. It's very important.

```suggestion
        chalk.bold(
```